python/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -82,47 +82,65 @@ def test_is_correct_type(self):
     def test_matches_fixture(self):
         self.assertEqual(self._fixtures["info"], self._manager.info())
 
-class Test_setSettings(FixtureAugmentedTestCase):
+
+class Test_settings(FixtureAugmentedTestCase):
+    """
+    Check plugin's implementation of managerApi.ManagerInterface.settings.
+    """
+
+    def test_when_retrieved_settings_modified_then_newly_queried_settings_unmodified(self):
+        original = self._manager.settings()
+        expected = original.copy()
+        original["update"] = 123
+        self.assertEqual(self._manager.settings(), expected)
+
+
+class Test_initialize(FixtureAugmentedTestCase):
     """
     Check plugin's implementation of
-    managerApi.ManagerInterface.setSettings.
+    managerApi.ManagerInterface.initialize.
     """
     def setUp(self):
-        self.collectRequiredFixture('some_settings_with_valid_keys', skipTestIfMissing=True)
+        super(Test_initialize, self).setUp()
+        self.__original_settings = self._manager.settings()
+
+    def tearDown(self):
+        self._manager.initialize(self.__original_settings)
+        super(Test_initialize, self).tearDown()
+
+    def test_empty_settings_succeeds(self):
+        self._manager.initialize({})
 
     def test_valid_settings_succeeds(self):
-        self._manager.setSettings(self.some_settings_with_valid_keys)
+        some_settings_with_valid_keys = self.requireFixture(
+            "some_settings_with_valid_keys", skipTestIfMissing=True)
 
-    def test_unknown_settings_keys_raise_KeyError(self):
-        unknown_settings = self.requireFixture("some_settings_with_invalid_keys")
-        with self.assertRaises(KeyError):
-            self._manager.setSettings(unknown_settings)
+        self._manager.initialize(some_settings_with_valid_keys)
 
+    def test_when_settings_have_invalid_keys_then_raises_KeyError(self):
+        unknown_settings = self.requireFixture(
+            "some_settings_with_invalid_keys", skipTestIfMissing=True)
 
-class Test_settings(FixtureAugmentedTestCase):
-    """
-    Check plugin's implementation of managerApi.ManagerInterface.settings.
-    """
+        with self.assertRaises(KeyError):
+            self._manager.initialize(unknown_settings)
 
-    def test_when_set_then_get_returns_updated_settings(self):
+    def test_when_settings_expanded_then_manager_settings_updated(self):
         updated = self.requireFixture("some_new_settings_with_all_keys", skipTestIfMissing=True)
-        self._manager.setSettings(updated)
+
+        self._manager.initialize(updated)
+
         self.assertEqual(self._manager.settings(), updated)
 
-    def test_when_set_with_subset_then_other_settings_unchanged(self):
+    def test_when_subset_of_settings_modified_then_other_settings_unchanged(self):
```

Comment:
[consider] How about `test_when_settings_have_subset_keys_then_other_settings_unchanged` to match the other names?

Reply:
Much better, updated to your suggestion in 28b67cfa